test/SlowTests/Utils/RavenDocumentsByEntityName.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Abstractions.Indexing;
+using Raven.Client.Indexes;
+using Raven.Client.Indexing;
+
+namespace SlowTests.Utils
+{
+    ///<summary>
+    /// Create an index that allows to tag entities by their entity name
+    ///</summary>
+    public class RavenDocumentsByEntityName : AbstractIndexCreationTask
```

Comment:
It should be removed. We don't have this index any longer, and we shouldn't have any tests relying on it.